std/algorithm/iteration.d

```diff
@@ -2898,9 +2898,10 @@ Params:
         element(s) to serve as separators in the joined range.
 
 Returns:
-A range of elements in the joined range. This will be a forward range if
-both outer and inner ranges of `RoR` are forward ranges; otherwise it will
-be only an input range. The
+A range of elements in the joined range. This will be a bidirectional range if
+both outer and inner ranges of `RoR` are at least bidirectional ranges. Else if
+both outer and inner ranges of `RoR` are forward ranges, the returned range will
+be likewise. Otherwise it will be only an input range. The
```

Comment:
I think that a note should be added to explain the behavior of the separator in the presence of bidirectional ranges.

Reply:
done